CourseGrab/ViewControllers/Search/SearchDetailCardTableViewCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SearchDetailCardTableViewCell.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 1/30/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SearchDetailCardTableViewCell: UITableViewCell {
+
+    private let cardView = UIView()
+    private let subtitleLabel = UILabel()
+    private let titleLabel = UILabel()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        isUserInteractionEnabled = false
+        setupCardView()
+        setupSubtitleLabel()
+        setupTitleLabel()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupCardView() {
+        cardView.layer.cornerRadius = 4
+        cardView.layer.borderWidth = 1
+        cardView.layer.borderColor = UIColor.courseGrabBorder.cgColor
+        cardView.backgroundColor = .white
+        contentView.addSubview(cardView)
+
+        cardView.snp.makeConstraints { make in
+            make.right.equalToSuperview().offset(-20)
+            make.left.equalToSuperview().offset(20)
```

Comment:
we should stick to using `leading` and `trailing` instead of `left` and `right`. i believe you can also condense these two lines into:
```
make.leading.trailing.equalToSuperview().inset(20)
```